tests/test_ecs/test_ecs_boto3.py

```diff
@@ -2015,3 +2015,62 @@ def _fetch_container_instance_resources(container_instance_description):
     registered_resources['PORTS'] = \
     [x['stringSetValue'] for x in registered_resources_list if x['name'] == 'PORTS'][0]
     return remaining_resources, registered_resources
+
+
+@mock_ecs
+def test_create_service_load_balancing():
+    client = boto3.client('ecs', region_name='us-east-1')
+    _ = client.create_cluster(
```

Comment:
More stylistc, but no need to assign to _ if its not going to be used.

Reply:
Completely right!